### PR TITLE
New deletion feature for article fragments

### PR DESCRIPTION
**Pull request by c-Rolland**
## Subject

I am targeting this branch, because i propose a new article fragments deletion feature. This feature allows the cancellations of removed fragments.

Closes #{put_issue_number_here}

## Changelog

```markdown
### Added
- Added to fragmentList.jquery.js `moveSelectedFragmentElements()` to move selected fragment in tmp dom
- Added to fragmentList.jquery.js `cancelFragmentDeletion()`  to cancel the fragment deletion
- Added to fragmentList.jquery.js `getFirstElement()` to get the first item form a list 

### Changed
  Cancellation of the fragment deletion.
### Deprecated

### Removed

### Fixed

### Security
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 /vendor/
 /.php_cs.cache
 /composer.lock
+.DS_Store
```
Comment (OskarStark) [inline .gitignore]: please remove this

File: src/Resources/public/js/fragmentList.jquery.js [hunk 2]
```diff
@@ -182,33 +186,104 @@
                 $form = this.getFormByFragmentId(id),
                 $cb = $form.find(this.options.formRemoveName);
 
+            //console.log(id);
```
Comment (OskarStark) [inline src/Resources/public/js/fragmentList.jquery.js]: please remove this

**Review by OskarStark — CHANGES_REQUESTED** [review]

**Comment by OskarStark** [comment]
- [x] Please fill the PR title and name what feature it is
- [x] Please fill the PR header accordingly

**Comment by greg0ire** [comment]
Please write a changelog directed at end users. Keep in mind that they probably do not care what file or class you changed, but what features you provided / what bugs you fixed. Try to answer the question "What benefit do I get from upgrading?"
More on http://keepachangelog.com

**Pull request closed without merging**